packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -363,6 +369,7 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onRenderAvatar,
       showMuteIndicator,
       strings,
+      /* @conditional-compile-remove(pinned-participants) */ drawerMenuHostId,
```

Comment:
Why is this conditionally compiled while it's usage is not?

Reply:
Good point. It's unneeded. Lemme remove it :)